src/filters/index.js

```diff
@@ -1,55 +1,68 @@
 // @ts-check
+
 /**
- * @param {string} str
- * @return {string}
+ * Uppercase the first character of a string.
+ *
+ * @param  {string} str String to uppercase the first letter of
+ * @return {string}     Original string with the first letter in uppercase
  */
 export const capitalize = str => str[0].toUpperCase() + str.slice(1);
 
 /**
- * @param {number} str
- * @param {string} currency
- * @param {string} [locale = navigator.language]
- * @return {string}
+ * Formats a number as currency using the user's locale. 1234.56 => '$1,234.56'
+ *
+ * @param  {number} num      The number to be formatted
```

Comment:
Originally this was written to accept a string because Vue directives will receive their value as a string. Technically it should be able to accept both. Nothing needs to change in the actual function because Intl will take either, but if we're going to change the input type, it should be `(string|number)` I think.